src/modules/companies/controller.js

```diff
@@ -44,4 +50,39 @@ const getReturns = async (request, h) => {
   return returns.map(rowMapper);
 };
 
+/**
+ * Gets a company for the specified ID
+ */
+const getCompany = async (request, h) => {
+  const { companyId } = request.params;
+  const company = await crmConnectors.companies.getCompany(companyId);
+  if (company) return company;
+  throw Boom.notFound(`Company ${companyId} not found`);
+};
+
+/**
+ * Gets all addresses for a company
+ */
+const getCompanyAddresses = async (request, h) => {
+  const { companyId } = request.params;
+  const companyAddresses = await crmConnectors.companies.getCompanyAddresses(companyId);
+  if (!isEmpty(companyAddresses)) return companyAddresses;
+  throw Boom.notFound(`Addresses for company ${companyId} not found`);
```

Comment:
If addresses is an empty array I would expect this to be returned with a `200`

I'd only expect `404` in the case where the company is not found